### PR TITLE
Travis CI setup

### PR DESCRIPTION
**Pull request by rimenes**

### CODE REVIEW

**Comment by rimenes** [comment]
Ready to merge. Ruby compatibility issues gonna be fixed in other PRs

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,4 @@
 source 'https://rubygems.org'
-ruby '2.0.0'
```
Comment (lucianot) [inline Gemfile]: @rimenes 
What's the idea behind removing the Ruby versioning?

Comment (rimenes) [inline Gemfile]: As Travis CI is set to test the project using different ruby versions it complains when the ruby version differ from the one set in the Gemfile.

"Your Ruby version is 1.8.7, but your Gemfile specified 2.0.0"

Not sure if it's the best solution

Comment (lucianot) [inline Gemfile]: http://docs.travis-ci.com/user/languages/ruby/

It seems we can set the Ruby version on `.travis.yml` or `.ruby-version`. I think that is a better alternative, no?

Comment (rimenes) [inline Gemfile]: We already have it set on both files. Another fix would put it back in the Gemfile and test only against this same ruby version. The Catarse project (https://github.com/catarse/catarse) does this way. What do you think?

Comment (lucianot) [inline Gemfile]: Yes, I think we should have a specific Ruby version. It's hard to guarantee compatibility with different versions and I don't see that as an important feature.

Comment (rimenes) [inline Gemfile]: Done

**Comment by lucianot** [comment]
@rimenes 
Merged. I also upgraded Ruby version to 2.1.5.

**Pull request merged**